README.md

```diff
@@ -1,2 +1,16 @@
-# github-maintenance
-Maintenance Scripts for GitHub Repositories
+# About
+
+This repo purpose is to gather useful scripts for different sporadic maintenance operations on GitHub repositories.
+
+# Scripts
+
+## bulk-update-github
+
+This script is to replace a string on every file under a certain directory on the default branch of all non-archived repositories in a GitHub organization.
+
+Run the following commands to execute the script:
+```
+cd bulk-update-github
+docker build -t bulk-update-github .
+docker run --rm -t bulk-update-github python bulk-update-github.py <GITHUB_ORG> <GITHUB_TKN> <DIR_LOOKUP> <STRING_OLD> <STRING_NEW> <MSG_COMMIT>
+```
```

Comment:
I wouldn't invest too long into it, but positional arguments arguments tend to be easy to misuse. Named arguments are better. Any popular programming language that is used on the command line will have at least one robust library in their ecosystem to handle this.

Python has click which I've used in the past and is very good: https://click.palletsprojects.com/en/8.1.x/

Reply:
Alternatively, the standard library also has some tools to handle this:
https://docs.python.org/3/library/argparse.html

Might be worth considering as an alternative if you don't want to overkill it with a third-party library for a simple use-case.